Source/WebCore/html/track/VTTCue.cpp

```diff
@@ -260,7 +260,7 @@ void VTTCueBox::applyCSSProperties()
 
     if (!cue->snapToLines()) {
         setInlineStyleProperty(CSSPropertyWhiteSpaceCollapse, CSSValuePreserve);
-        setInlineStyleProperty(CSSPropertyTextWrap, CSSValueNowrap);
+        setInlineStyleProperty(CSSPropertyTextWrapMode, CSSValueNowrap);
```

Comment:
Just confirming that we do not need to also set `text-wrap-style` here?

Reply:
Same here